rosa_getting_started/rosa-sts-required-aws-service-quotas.adoc

```diff
@@ -0,0 +1,12 @@
+include::modules/attributes-openshift-dedicated.adoc[]
+[id="rosa-sts-required-aws-service-quotas"]
+= Required AWS service quotas
+:context: rosa-sts-required-aws-service-quotas
+toc::[]
+
+Review this list of the required Amazon Web Service (AWS) service quotas that are required to run an {product-title} cluster.
+
+include::modules/rosa-required-aws-service-quotas.adoc[leveloffset=+1]
```

Comment:
The table of the required quota is so large in the doc. Is it what we expected? Create a rosa cluster doesn't need so many resources, here is a doc of OCP describes how many quota will be reserved for a cluster creation. Would anybody help to double confirm the accurate of the data?
https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-account.html#installation-aws-limits_installing-aws-account